packages/badge/src/Badge.ts

```diff
@@ -51,6 +51,21 @@ export class Badge extends SizedMixin(ObserveSlotText(SpectrumElement, '')) {
     public variant: BadgeVariant = 'informative';
 
     protected override render(): TemplateResult {
+        if (window.__swc.DEBUG) {
```

Comment:
What if we alias'ed this in SpectrumBase eventually so components could do a `this.DEBUG_MODE` or something? Then if you want to alter the way we set and fetch the state, we could do that at SpectrumBase without having to update every component file.